### PR TITLE
EDUCATOR-1086  Allow range of hours/units or one, non-range value in 'Estimated Effort' field

### PR DESCRIPTION
**Pull request by Rabia23**
### Description
In the past the 'Estimated Effort' field in Publisher requires that course teams submit a range of hours/units but now it will allow range of hours/units or one, non-range value in course runs. In this case the minimum effort field is considered.

### How to Test?
**Sandbox**
https://discovery-educator-973.sandbox.edx.org/publisher/course_runs/1/
See the `Estimated Effort` field in **All** and **Drupal** tab.

### Screenshots
**Before Fix**
before-fix-1 2017-08-11 at 4 35 26 pm

before-fix-2 2017-08-11 at 4 35 45 pm

**After Fix**
after-fix-1 2017-08-11 at 4 37 32 pm

after-fix-2 2017-08-11 at 4 37 54 pm

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x]  @asadazam93 

FYI: @awaisdar001 

### Post-review
- [ ] Rebase and squash commits

### CODE REVIEW

**Review by asadazam93 — APPROVED** [review]
Looks good

File: course_discovery/apps/publisher/tests/test_forms.py [hunk 1]
```diff
@@ -101,6 +101,32 @@ def test_minimum_effort(self):
...
+
+    def test_minimum__effort_is_not_empty(self):
+        """
+        Verify that 'clean' raises 'ValidationError' error if Minimum effort is
+        empty.
+        """
+        run_form = CustomCourseRunForm()
+        run_form.cleaned_data = {'max_effort': 4}
+        with self.assertRaises(ValidationError):
+            run_form.clean()
```
Comment (awaisdar001) [inline course_discovery/apps/publisher/tests/test_forms.py]: there are many `ValidationError` in the form `clean` method, We should also test if the expected `ValidationError` has occurred.

File: course_discovery/apps/publisher/models.py [hunk 2]
```diff
@@ -725,7 +725,7 @@ def can_send_for_review(self):
             course_run.pacing_type, course_run.has_valid_staff, course_run.is_valid_micromasters,
             course_run.is_valid_professional_certificate, course_run.is_valid_xseries, course_run.language,
             course_run.transcript_languages.all(), course_run.lms_course_id, course_run.min_effort,
-            course_run.max_effort, course_run.video_language, course_run.length
```
Comment (awaisdar001) [inline course_discovery/apps/publisher/models.py:728]: We are updating `can_send_for_review` method in the `CourseRunState` model, the relevant test is missing.

**Review by awaisdar001 — CHANGES_REQUESTED** [review]

File: course_discovery/apps/publisher/models.py (hunk 2 above)
Comment (Rabia23) [reply course_discovery/apps/publisher/models.py:728]: @awaisdar001 relevant test exists. It has been updated already. 
Here is a link:
https://github.com/edx/course-discovery/pull/1087/files#diff-7541c140671314e6639d6a0fe3f4c4df

File: course_discovery/apps/publisher/tests/test_forms.py (hunk 1 above)
Comment (Rabia23) [reply course_discovery/apps/publisher/tests/test_forms.py]: Tests already exist for the other `ValidationError` in the form `clean` method.

**Pull request merged**